src/features/metadata-instance-editor/__tests__/metadataUtil-test.js

```diff
@@ -1,102 +1,125 @@
+        {
+            object: {
+                id: '5',
+                type: 'date',
+                key: 'field 5',
+                displayName: 'field 5',
+                isHidden: true,
+            },
+            description: 'Hidden Field 5',
+            expected: true,
         },
-        description: 'Hidden Field 5',
-        expected: true,
-    },
-    {
-        object: {
-            id: '6',
-            type: 'date',
-            key: 'field 6',
-            displayName: 'field 6',
-            hidden: true,
+        {
+            object: {
+                id: '6',
+                type: 'date',
+                key: 'field 6',
+                displayName: 'field 6',
+                hidden: true,
+            },
+            description: 'Hidden Field 6',
+            expected: true,
         },
-        description: 'Hidden Field 6',
-        expected: true,
-    },
-    {
-        object: {
-            id: '7',
-            type: 'date',
-            key: 'field 7',
-            displayName: 'field 7',
-            isHidden: false,
+        {
+            object: {
+                id: '7',
+                type: 'date',
+                key: 'field 7',
+                displayName: 'field 7',
+                isHidden: false,
+            },
+            description: 'Visible Field 7',
+            expected: false,
         },
-        description: 'Visible Field 7',
-        expected: false,
-    },
-    {
-        object: {
-            id: '8',
-            type: 'date',
-            key: 'field 8',
-            displayName: 'field 8',
-            isHidden: false,
+        {
+            object: {
+                id: '8',
+                type: 'date',
+                key: 'field 8',
+                displayName: 'field 8',
+                isHidden: false,
+            },
+            description: 'Visible Field 8',
+            expected: false,
         },
-        description: 'Visible Field 8',
-        expected: false,
-    },
-].forEach(({ description, object, expected }) => {
-    test(description, () => {
-        const actual = isHidden(object);
-        expect(actual).toEqual(expected);
+    ].forEach(({ description, object, expected }) => {
+        test(description, () => {
+            const actual = isHidden(object);
+            expect(actual).toEqual(expected);
+        });
+    });
+});
+
+describe('convertTemplateFilters()', () => {
+    const singleFilter = 'abcde';
+    const multipleFilters = ['ghijk', 'lmnop', 'uvxyz'];
+    [
+        {
+            filters: singleFilter,
+            description: 'should convert a single filter into a Set object with one item',
+            expected: new Set([singleFilter]),
+        },
+        {
+            filters: multipleFilters,
+            description: 'should convert multiple template filters into a Set object',
+            expected: new Set(multipleFilters),
+        },
+    ].forEach(({ description, filters, expected }) => {
```

Comment:
We don't really use this style of testing as much now that we've moved away from Karma. Can we convert this to use `jest.each`?